### PR TITLE
hotfix: wrong export paths, hasAll method and tests

### PR DESCRIPTION
**Pull request by mateusmcordeiro**
After the awsome update we`ve done, i tried to install the library inside my project and for my surprise, it was not working hahaha, but i saw it was a easy thing to solve locally and this PR is to solve this for the others users.

it was just a configuration missing on package.json and it is going to need to push a new patch version to npm.

i dont know why no one has opened a issue about it, because i saw the last version number of downloads on npm and its like 60+.

------

I`ve pushed some more code fixing about the hasAll method and added some tests.

### CODE REVIEW

File: src/lib/query.ts [hunk 1]
```diff
@@ -157,7 +157,7 @@ export class Query {
   hasAll(value: any[]) {
     if (this._expression)
       this._expression.template =
-        "%not(%term && %value?.every(value => %term.includes?.(value))";
+        "%not(%term && %value?.every(value => %term.includes?.(...value)))";
```
Comment (jacwright) [inline src/lib/query.ts:160]: This is incorrect. Here is the simplified code that this turns into before this fix:

```js
function hasAll(obj: any, prop: string, value: any[]) {
  return value.every(item => obj[prop].includes(item));
}
```

To make it more readable, you can change `value => %term.includes?.(value)` to `item => %term.includes?.(item)`, but other than that, it should be left as-is.

File: __tests__/query.spec.js [hunk 2]
```diff
@@ -72,11 +123,29 @@ describe("Query class tests", () => {
...
+
+    const result = myQuery.on(objects).length === objects.length;
+
+    expect(result).toEqual(true);
+  });
+
+  it("should hasAll work with array", () => {
+    const juicyAndSweetFruits = objects.filter((fruit) =>
+      fruit.characteristics.includes(["Juicy", "Sweet"])
     );
```
Comment (jacwright) [inline __tests__/query.spec.js:143]: This is incorrect. It is giving back an empty array and should be:
```js
fruit.characteristics.includes("Juicy", "Sweet")
```
Once you fix this and change `hasAll` back to the way it was, the tests will pass.

**Review by jacwright — CHANGES_REQUESTED** [review]
Apologies for the bad package.json. Thank you!

The `hasAll` was correct. The tests are a great add, though one of them is incorrect.

File: src/lib/query.ts (hunk 1 above)
Comment (mateusmcordeiro) [reply src/lib/query.ts]: i made the tests first, as you saw there i wrote some includes in a  wrong way on the test and i didnt saw it. 

About this, im going to remove this spread on the value.

But it was missing a `)` at the end, this change is right and i will not change it.

**Comment by jacwright** [comment]
Published as 0.2.2

**Pull request merged**